mozilla/blurts-server
# Add plans to the landing page on wide screens

# References: 
Jira: MNTOR-2471
Figma: https://www.figma.com/file/YqpVpUbSTzZMj9Jbp2po8Q/Mozilla-Monitor-Premium-LP?type=design&node-id=1942-21098&mode=dev




# Description

This adds the plans comparison table.

Version for smaller screens will be added later.

# How to test

Make sure the tooltips open, the pricing toggle changes what's shown and where the buttons go to, and that it's accessible.

# Checklist (Definition of Done)
- [ ] All acceptance criteria are met. - Mobile view still to come
